packages/engine-multi/src/api/execute.ts

```diff
@@ -49,10 +48,10 @@ const execute = async (context: ExecutionContext) => {
     const whitelist = options.whitelist?.map((w) => w.toString());
 
     const runOptions = {
-      adaptorPaths,
```

Comment:
I am still not sure that replacing adaptorPaths with repoDir is the right answer.

First off, adaptorPaths will run more quickly. If we pass in the repoDir then the runtime has to do a lot more work to load the correct module (including parsing the package json).

Secondly, this fix is kind of a workaround a bug in the runtime, where paths get blatted.

I think a better approach here would be for autoinstall to write tothe plan a path to the correct, versioned adatpor. So the plan changes. The runtime receives neither adaptorpaths nor repodir, but absolute paths in the plan